src/prefect/server/events/models/automations.py

```diff
@@ -92,6 +93,9 @@ async def read_automation_by_id(
 
 
 async def _notify(session: AsyncSession, automation: Automation, event: str):
+    if not PREFECT_API_SERVICES_TRIGGERS_ENABLED.value():
```

Comment:
fwiw [we don't need](https://github.com/PrefectHQ/prefect/blob/c7debf1d256a192a0da572b6ed62c01edd74ad1f/src/prefect/settings.py#L248-L252) `.value()` for truthy checks on settings

Reply:
Whoa thanks, I hadn't caught that detail